### PR TITLE
Use dd network tracer to resolve DNAT

### PR DESCRIPTION
**Pull request by Q69K**

### CODE REVIEW

File: checks/net.go [hunk 1]
```diff
@@ -5,7 +5,13 @@ import (
 	"errors"
 	"fmt"
 	"github.com/StackVista/stackstate-agent/pkg/aggregator"
-	"strconv"
+	"github.com/StackVista/stackstate-agent/pkg/ebpf"
```
Comment (craffit) [inline checks/net.go:8]: this branch depending on a stackstate-agent version from 14th of november, best to go to the latest master before bumping (see go.mod)

File: checks/net_linux.go [hunk 2]
```diff
@@ -5,42 +5,41 @@ package checks
...
 		if proc := os.Getenv("HOST_PROC"); proc != "" {
 			conf.ProcRoot = proc
 		}
-		conf.MaxConnections = cfg.NetworkTracerMaxConnections
-		conf.BackfillFromProc = cfg.NetworkInitialConnectionsFromProc
-		conf.EnableTracepipeLogging = cfg.NetworkTracer.EbpfDebuglogEnabled
-		conf.HttpMetricConfig = *cfg.NetworkTracer.HTTPMetrics
-		conf.EnableProtocolInspection = cfg.NetworkTracer.EnableProtocolInspection
+		conf.MaxTrackedConnections = uint(cfg.NetworkTracerMaxConnections) // TODO make sure it is the same
+		conf.EnableConntrack = true
```
Comment (craffit) [inline checks/net_linux.go:39]: does using conntrack require additional permissions on k8s? maybe on openshift?

File: go.mod [hunk 3]
```diff
@@ -11,218 +11,297 @@ replace (
...
-	github.com/DataDog/gopsutil v0.0.0-20200624212600-1b53412ef321
-	github.com/DataDog/sketches-go v1.1.0
-	github.com/DataDog/zstd v0.0.0-20160706220725-2bf71ec48360
-	github.com/StackExchange/wmi v0.0.0-20181212234831-e0a55b97c705
-	github.com/StackVista/stackstate-agent v0.0.0-20221114140924-00b602b70d2a
+	github.com/DataDog/gopsutil v0.0.0-20211112180027-9aa392ae181a
+	github.com/DataDog/sketches-go v1.2.1
+	github.com/DataDog/zstd v1.4.8
+	github.com/StackExchange/wmi v1.2.1
+	github.com/StackVista/stackstate-agent v0.0.0-20221214135916-7cec813b9442
```
Comment (craffit) [inline go.mod:32]: i did not see an mr for this guy

File: checks/net_linux.go (hunk 2 above)
Comment (Q69K) [reply checks/net_linux.go:39]: No, this is also just eBPF hooks

File: go.mod (hunk 3 above)
Comment (Q69K) [reply go.mod]: https://gitlab.com/stackvista/agent/stackstate-agent/-/merge_requests/203

**Review by craffit — APPROVED** [review]

**Pull request merged**